### PR TITLE
new release info (0.12.0)

### PR DESCRIPTION
**Pull request by skoudoro**
commit summarise from 0.11 to 0.12.0

### CODE REVIEW

**Comment by Garyfallidis** [comment]
Cool, thanks @skoudoro

File: doc/release0.12.rst [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+* Kesshi Jordan
+* Manu Tej Sharma
+* Marc-Alexandre Côté
+* Matthew Brett
+* Matthieu Dumont
+* Nil Goyette
+* Omar Ocegueda Gonzalez
+* Rafael Henriques
+* Rafael Neto Henriques
+* RafaelNH
```
Comment (arokem) [inline doc/release0.12.rst]: Rafael is duplicated.

File: doc/release0.12.rst [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+* Sourav Singh
+* Stephan Meesters
+* StongeEtienne
+* Theaud
+* Tingyi Wanyan
+* Tom Wright
+* Vibhatha Abeykoon
+* Yaroslav Halchenko
+* algo
+* etpeterson
```
Comment (arokem) [inline doc/release0.12.rst]: Maybe replace this with "Eric Peterson" (considering we know who this is 😄 )?

**Comment by skoudoro** [comment]
Thank you for this review @arokem. we correct this file on [this commit](https://github.com/nipy/dipy/pull/1275/commits/df4a717fa7c49a6b2c8de0f770c9dd5b4d9921d8) (PR #1275)

**Pull request merged**